### ISSUE TITLE
Practice Mode Conc Abuse

### ISSUE THREAD

**Issue by NinjaTheCookie** [momentum-mod/game#1168]
### **Abuse 1:**
It is possible to cook a Conc grenade while in the effects of Practice Mode. 

When entering Practice Mode, the timer continues running, however this freezes you in-place with all your momentum stored, letting you wait for a grenade to cook and explode right after the other continuously, allowing normally impossible strategies to be achieved.

**Example of issue:**
![concabuse](https://user-images.githubusercontent.com/62808028/100149123-9cc39400-2e95-11eb-96b0-538bf3a01e99.gif)

**Comment by Gocnak** [comment]
Abuse 1 is resolvable by making the conc grenade a CMomExplosive, and should be fixed by 0.8.7.

The second issue is itself another problem and should be pulled out to its own card. Unless things are directly related (one bug causes the next), you want to keep issues pretty modular so they can be tackled in parallel.

**Comment by NinjaTheCookie** [comment]
> Abuse 1 is resolvable by making the conc grenade a CMomExplosive, and should be fixed by 0.8.7.
> 
> The second issue is itself another problem and should be pulled out to its own card. Unless things are directly related (one bug causes the next), you want to keep issues pretty modular so they can be tackled in parallel.

@Gocnak 
The main problem I bring up in this issue is that the world continues around the player while frozen in-run and this can be abused in multiple ways, these are just examples of ways.

**Comment by NinjaTheCookie** [comment]
**Issue was changed to be more focused on Conc, I will do a separated more in-depth issue on world entity issues involving Practice Mode**
